### PR TITLE
Eclipse Mars Updates

### PR DESCRIPTION
**Pull request by drkstr101**
These updates should bring the project up to date with Eclipse Mars 2 milestone release.

Most of the file changes were done automatically by Eclipse. The actual changes I made were:

1. Java version has been set to 1.8 on all projects
2. An unresolved (and unused) dependency has been removed from the test projects
3. All minimum dependency versions have been pinned to the Eclipse Mars release versions

### CODE REVIEW

File: org.agileware.natural.cucumber.ui/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
- org.agileware.natural.cucumber.ui.contentassist.antlr,
- org.agileware.natural.cucumber.ui.internal,
- org.agileware.natural.cucumber.ui.contentassist,
- org.agileware.natural.cucumber.ui.quickfix
-Bundle-Activator: org.agileware.natural.cucumber.ui.CucumberActivator
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.agileware.natural.cucumber.ui
+Bundle-Vendor: Roberto Lo Giacco
+Bundle-Version: 1.3.5.qualifier
```
Comment (drkstr101) [inline org.agileware.natural.cucumber.ui/META-INF/MANIFEST.MF:5]: missing version bump

File: org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/CucumberGeneratorMWE.mwe2 [hunk 2]
```diff
@@ -0,0 +1,31 @@
+module org.agileware.natural.cucumber.generator.CucumberGeneratorMWE
```
Comment (drkstr101) [inline org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/CucumberGeneratorMWE.mwe2:1]: This was auto-generated by eclipse. Should it be removed?

File: org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/Main.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/*
```
Comment (drkstr101) [inline org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/Main.java:1]: This was auto-generated by eclipse. Should it be removed?

File: org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/CucumberGeneratorMWE.mwe2 (hunk 2 above)
Comment (rlogiacco) [reply org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/CucumberGeneratorMWE.mwe2:1]: no need to remove it

**Pull request merged**